tools/KoreBuild.Console/Commands/DockerFiles/debian.dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM microsoft/dotnet:2.0-runtime-deps-jessie
```

Comment:
Stretch doesn't work yet due to [this](https://github.com/dotnet/cli/blob/master/scripts/obtain/dotnet-install.sh#L67), so it can't be our default, but I'll rename it for future-proofing.